dev/com.ibm.ws.jca_fat_bvt/publish/servers/com.ibm.ws.jca.fat.bvt/server.xml

```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2011,2022 IBM Corporation and others.
+    Copyright (c) 2011,2022,2025 IBM Corporation and others.
```

Comment:
the header should be `2011,2025`

for future reference, the copyright header should only have 1 or 2 years.

if new file or file modified the same year it was created, then just the current year's date.
if file modified in a year different it was created, then `<created-year>, <modified-year>`.